docs/changehistory/NextVersion.md

```diff
@@ -45,6 +46,7 @@ Table of contents:
 ### Additions
 
 - `ConditionalIconItem.isConditionalIconItem` is now a `ConditionalIconItem` _type predicate_, on top of returning a boolean.
+- `Dialog` now has resize functionality from iTwinUI `Dialog`.
```

Comment:
Move under `### Changes`. Reword to: "Changed `Dialog` to use resize..."?

Reply:
Done.